### ISSUE TITLE
Syncing fields issue: Unable to save form SQLSTATE[42000] 1118 Row size too large

### ISSUE THREAD

**Issue by gddotorg** [verbb/formie#1064]
### Describe the bug

We are facing an issue, when saving forms with synced fields. 
If I create a new form and add existing fields, everything still works as expected. Unfortunately, I cannot say for sure when the error occurs. However, our customer reports that the following error message appears when they want to adjust the form. I then test it myself and also get the error message. 

`Unable to save form: {"general":["SQLSTATE[42000]: Syntax error or access violation: 1118 Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs\nThe SQL being executed was: ALTER TABLE 'cc_fmc_contactformnaasen' ADD 'field_Salutation_awwhcecm' varchar(255)"]}.`

The error also occurs with other fields, so it is not exclusively due to one field. I took a look at the tables and found that many columns exist more than once. So field_Salutation_zffutjbx, field_Salutation_qirxokez, field_Salutation_vdwlfuud and so on. 

If you need more information from me, let me know.

Best regards,
Georg		

### Steps to reproduce

1. Add new form with existing sync fields.
2. Saving form does work.
3. Resave after a while occurs the error.

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

3.7.44

### Plugin version

1.6.10

### Multi-site?

Yes

**Comment by engram-design** [comment]
This usually points to a form with too many fields for MySQL to handle. How many fields does your `contactformnaasen` for have? How many columns are there in the `fmc_contactformnaasen` database table?

**Comment by gddotorg** [comment]
`contactformnaasen` has 9 fields and `fmc_contactformnaasen` has 122 `field_` columns.

**Comment by engram-design** [comment]
Something certainly seems off there, 122 fields would be reaching the MySQL limit, so that makes sense that it's firing this error, but based on the number of fields used in this form, that number is way off.

I've just added a new console command to cleanup content table that get in this sort of state. This will look at all the fields in your form, check if there's any extra field columns and remove those if they don't belong to an existing field. Formie should do this automatically whenever you save a form.

```
./craft formie/gc/prune-content-table-fields
```

To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.11](https://github.com/verbb/formie/releases/tag/1.6.11)
